src/components/director-view/director-view.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import axios from 'axios';
+
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Spinner from 'react-bootstrap/Spinner';
+
+import { MovieCard } from '../movie-card/movie-card';
+
+import './director-view.scss';
+import { Link } from 'react-router-dom';
+
+export class DirectorView extends React.Component {
+    constructor() {
+        super()
+
+        this.state = {
+            movies: [],
+            directorMovies: []
+        };
+    }
+
+    getMovies(token) {
+        console.log("Loading movies from API...");

```

Comment:
You can add an explicit loading state here since heroku takes some time to load initially. `this.setState({ loading: true })`

Also the loader you are using needs to be styled, I suggest centering it, using absolute positioning techniques.
![Screen Shot 2021-01-31 at 4 22 40 PM](https://user-images.githubusercontent.com/15272175/106398420-a7902d00-63e0-11eb-9d09-2f2a1e462e67.png)